configure.ac

```diff
@@ -302,6 +302,17 @@ AC_ARG_WITH([cc-image],
     ]
 )
 
+AC_ARG_ENABLE([autogopath],
+	AS_HELP_STRING([--enable-autogopath], [enable autogopath @<:@default=false@:>@]),
```

Comment:
@jodh-intel This option will help to automate builds on OBS, OBS does not have GOPATH defined and it has to be defined, and prepare the environment to build the package.